### PR TITLE
Adding scale in styles branch

### PR DESCRIPTION
**Pull request by aurnabnil66**

### CODE REVIEW

File: App.tsx
Comment (SkrChowdhury) [inline App.tsx:1]: Hide Header from login screen

File: src/app/Screens/Login/styles.ts [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    transform: [{translateY: -15}], // Adjust the vertical alignment of the icon
+    zIndex: 1,
+  },
+  emailInput: {
+    width: scale(280),
+    height: scale(50),
+    backgroundColor: '#F3F2E9',
+    fontSize: moderateScale(14),
+    fontWeight: '600',
+    borderRadius: 16,
```
Comment (SkrChowdhury) [inline src/app/Screens/Login/styles.ts:57]: use size matters for padding also

File: src/app/Screens/Login/styles.ts [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    position: 'absolute',
+    left: scale(16),
+    top: '50%',
+    transform: [{translateY: -18}], // Adjust the vertical alignment of the icon
+    zIndex: 1,
+  },
+  showPasswordIcon: {
+    position: 'absolute',
+    right: scale(10),
+    top: '47%',
```
Comment (SkrChowdhury) [inline src/app/Screens/Login/styles.ts:75]: use size matters for top bottom also

File: src/app/Screens/MedicineDetails/MedicineDetails.tsx [hunk 3]
```diff
@@ -1,11 +1,15 @@
...
+import {useNavigation} from '@react-navigation/native';
 
 function MedicineDetails() {
+  const navigation = useNavigation();
   return (
     <View style={styles.container}>
-      <TouchableOpacity style={styles.button}>
+      <TouchableOpacity
+        style={styles.button}
+        onPress={() => navigation.navigate('Login' as never)}>
```
Comment (SkrChowdhury) [inline src/app/Screens/MedicineDetails/MedicineDetails.tsx:12]: create a function then use that function in onPress

File: src/app/Screens/MedicineDetails/styles.ts [hunk 4]
```diff
@@ -11,7 +12,7 @@ const styles = StyleSheet.create({
     width: 343,
```
Comment (SkrChowdhury) [inline src/app/Screens/MedicineDetails/styles.ts:12]: use size matters here

File: src/app/Screens/QrCode/styles.ts [hunk 5]
```diff
@@ -8,15 +9,15 @@ const styles = StyleSheet.create({
...
   },
   image: {
-    width: 270,
-    height: 270,
+    width: scale(230),
+    height: verticalScale(230),
     resizeMode: 'contain',
   },
   button: {
     width: 343,
```
Comment (SkrChowdhury) [inline src/app/Screens/QrCode/styles.ts:17]: use size matters here

File: src/assets/login-header-frame.png
Comment (SkrChowdhury) [inline src/assets/login-header-frame.png:1]: use View instead of image

**Review by SkrChowdhury — COMMENTED** [review]
please follow these reviews and make sure to fix them .

File: App.tsx
Comment (aurnabnil66) [reply App.tsx:1]: Done

File: src/app/Screens/Login/styles.ts (hunk 2 above)
Comment (aurnabnil66) [reply src/app/Screens/Login/styles.ts:75]: done

File: src/app/Screens/MedicineDetails/MedicineDetails.tsx (hunk 3 above)
Comment (aurnabnil66) [reply src/app/Screens/MedicineDetails/MedicineDetails.tsx:12]: I will do it later

File: src/app/Screens/QrCode/styles.ts (hunk 5 above)
Comment (aurnabnil66) [reply src/app/Screens/QrCode/styles.ts:17]: Done

**Pull request merged**